feat: repeater proxy (#160)
Escape-Technologies/docs: docs/03-platform/03-enterprise/07-repeater.mdx

    repeater_mtls: true
```

### Use a custom proxy to connect to escape

As this software use two types of connections : connection for the repeater to escape and connection from the repeater to your internal APIs, we ignore the `HTTP_PROXY` and `HTTPS_PROXY` environment variables.

If you want to connect to the escape tenant (`repeater.escape.tech`) via a proxy, you can use the `ESCAPE_REPEATER_PROXY_URL` environment variable.

```bash
docker run -it --rm --name escape-repeater \
    -e ESCAPE_REPEATER_PROXY_URL=http://user:pass@my-proxy.server.tld:1234 \
    -e ESCAPE_REPEATER_ID=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx \
    escapetech/repeater:latest
```

> Note that as we use gRPC, the proxy must support it.
> You so need an HTTP/2 proxy.

Reviewer: ```suggestion
Note that as we use gRPC, the proxy must support it, which implies your proxy MUST support HTTP/2.
```
